annotations/transport/annotations_router.go

```diff
@@ -106,13 +107,19 @@ func (h *AnnotationHandler) handleGetAnnotation(w http.ResponseWriter, r *http.R
 		return
 	}
 
-	a, err := h.annotationService.GetAnnotation(ctx, *id)
+	s, err := h.annotationService.GetAnnotation(ctx, *id)
 	if err != nil {
 		h.api.Err(w, r, err)
 		return
 	}
 
-	h.api.Respond(w, r, http.StatusOK, a)
+	c, err := storedAnnotationToEvent(s)
```

Comment:
Per the service changes above, the handler needs to convert the `StoredAnnotation` type to an `AnnotationEvent` before responding per the API contract.